src/dotnet/Users.Contracts/Backend/IUserProfilesBackend.cs

```diff
@@ -6,4 +6,10 @@ public interface IUserProfilesBackend
     Task<UserProfile?> Get(string userId, CancellationToken cancellationToken);
     [ComputeMethod(KeepAliveTime = 10)]
     Task<UserProfile?> GetByName(string name, CancellationToken cancellationToken);
+
+    [CommandHandler]
+    public Task UpdateStatus(UpdateStatusCommand command, CancellationToken ct = default);
```

Comment:
Please don't use shortcuts like `ct` in public parts of the API.

It's fine to use such shortcuts for locals, etc. though.

Reply:
done